readme.md

```diff
@@ -51,24 +51,49 @@ You can pass options to `awesome-typescript-loader` as a first argument
 // next.config.js
 const withAwesomeTypescript = require("next-awesome-typescript");
 
-const options = {
+const awesomeTypescriptOptions = {
   useCheckerPlugin: true,
   loaderOptions: {
     transpileOnly: false,
   },
 };
 
-module.exports = withAwesomeTypescript(options);
+module.exports = withAwesomeTypescript(awesomeTypescriptOptions);
```

Comment:
It should be `module.exports = withAwesomeTypescript({ awesomeTypescriptOptions });`